packages/hardhat/.gitignore

```diff
@@ -10,4 +10,3 @@ cache
 artifacts
 
 deployments
-
```

Comment:
You might want to leave this as it was, so there is no changes to this file

Reply:
this is how it was.. not sure how to remove the history

Reply:
Just add a new line in the file (as it was before) so It won't show on the diff.

![image](https://user-images.githubusercontent.com/2486142/197348470-b8bbbcb9-9288-4d56-8e18-5d28cb5e276b.png)

Reply:
So strange... I have the extra line in my editor and shows no changes.

<img width="693" alt="Screen Shot 2022-10-22 at 12 32 53 PM" src="https://user-images.githubusercontent.com/9419140/197350929-8821ecbb-6ea8-4eba-a219-54f463375997.png">

Reply:
Maybe your editor is configured to strip the last line when saving?

Anyway, no worries, I can update that. I'll review & test again on Monday.

Thanks Jason!